src/components/ExampleComponent/ExampleComponent.jsx

```diff
@@ -0,0 +1,87 @@
+// Required for all new components
+import _ from 'lodash';
+import React from 'react';
+import PropTypes from 'prop-types';
+import { lucidClassNames } from '../../util/style-helpers';
+import { createClass, findTypes, omitProps } from '../../util/component-types';
+
+import './ExampleComponent.less';
+const cx = lucidClassNames.bind('&-ExampleComponent');
+
+const { bool, func, node, string, object } = PropTypes;
+
+/**
+ * {"categories": ["mycategory"]}
+ *
+ * Update "mycategory" to set the category your component falls under. You can see available categories
+ * in the left hand column.
+ *
+ * Describe your component here. How it should be used, use cases and more.
+ * Please see examples for more information.
+ */
+const ExampleComponent = createClass({
+	displayName: 'ExampleComponent',
+	_isPrivate: true,
```

Comment:
I would remove this to make the example easier to find by searching
and it's not exposed in index.js so no real need to make it private IMO